admin-dev/themes/new-theme/js/pages/product/components/combination-modal/CombinationModal.vue

```diff
@@ -237,6 +237,10 @@
           input.addEventListener('change', () => {
             this.isFormUpdated = true;
           });
+
+          window.document.addEventListener('datepickerChange', () => {
```

Comment:
Shouldn't it watch the event on the document from the iframe? This one is the document from the parent right?